### DISCUSSION TITLE
Horizontal lines not rendering

### DISCUSSION THREAD

**Discussion by paulinestacey** [squidfunk/mkdocs-material#7450] [Q&A] [answered]
I've used "---" for a few years to create horizontal lines without issue. Recently noticed that they're not rendering. I've not changed anything (knowingly) and wonder if anyone else has seen this on their pages?

**Comment by alexvoss** [comment]
Are you talking about existing content not rendering as you would expect or new content? You need to make sure that the three dashes are preceded and followed by empty lines, see [the Markdown Guide](https://www.markdownguide.org/basic-syntax/#horizontal-rules). 

If that is not the issue then we'd need to look at your project, so a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) would be good that clearly shows what the issue is.

**Reply by paulinestacey** [reply]
Thanks for the response. It's existing content that has recently stopped rendering horizontal lines. (I've also tried adding new content and that doesn't render either.) The three dashes are preceded and followed by empty lines as per the Markdown Guide and have rendered with no issue for the last 3 years. Now the markdown renders as shown here:
![image](https://github.com/user-attachments/assets/0f8d8d27-8487-4c39-bc0e-604ff6cf6579)
![image](https://github.com/user-attachments/assets/fb3e31e6-e4da-471d-839b-e7e21a9dc992)

If there's no known issue, I'll create a minimal reproduction. Thanks.

**Reply by alexvoss** [reply]
I think we'd need a reproduction because the most minimal reproduction I could create did not reproduce the problem. Please make sure to follow the [minimal reproduction steps](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Reply by squidfunk** [reply, marked as answer]
But this is correct? There are horizontal lines in your screenshot? I'm not sure I understand the problem 😅

**Reply by paulinestacey** [reply]
Whoa! I think it's my main monitor giving out weirdness! Testing on different browsers on other monitors and even viewing this page with the screenshot shows the lines... but they're not visible on my main monitor! Time for a new monitor... and apologies for wasting your time with my non-existent problem!

**Reply by alexvoss** [reply]
@paulinestacey, I would say you found an interesting edge-case! I did not notice that the screenshot contains the lines - perhaps because they are faint and I was a bit tired...

**Reply by kamilkrzyskow** [reply]
> Time for a new monitor...  

I'm interested to know what the issue is in this case. The monitor not displaying colors the right way, so the contrast ratio was too small. Or the link was too thin, and some mix of not native resolution / scaling caused the line to not display properly. 🤔

**Reply by diffloc** [reply]
> Whoa! I think it's my main monitor giving out

Whoa! I just fought this same exact thing for over an hour. The issue? My monitor as well. Dual 32" 4k displays (same models). One shows it, the other doesn't (facepalm).
